[2024-03-12T11:13:27.300+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v02.insert_into_table manual__2024-03-12T11:13:25.089348+00:00 [queued]>
[2024-03-12T11:13:27.320+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v02.insert_into_table manual__2024-03-12T11:13:25.089348+00:00 [queued]>
[2024-03-12T11:13:27.321+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-12T11:13:27.347+0000] {taskinstance.py:2214} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-03-12 11:13:25.089348+00:00
[2024-03-12T11:13:27.358+0000] {standard_task_runner.py:60} INFO - Started process 2395 to run task
[2024-03-12T11:13:27.367+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v02', 'insert_into_table', 'manual__2024-03-12T11:13:25.089348+00:00', '--job-id', '889', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpoif9_382']
[2024-03-12T11:13:27.372+0000] {standard_task_runner.py:88} INFO - Job 889: Subtask insert_into_table
[2024-03-12T11:13:27.482+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v02.insert_into_table manual__2024-03-12T11:13:25.089348+00:00 [running]> on host 26e63b1d0d39
[2024-03-12T11:13:27.657+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='daniel' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v02' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-03-12T11:13:25.089348+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-12T11:13:25.089348+00:00'
[2024-03-12T11:13:27.660+0000] {sql.py:276} INFO - Executing:  
            insert into dag_runs (dt, dag_id) values ('2024-03-12', 'dag_with_postgres_operator_v02' )
        )
[2024-03-12T11:13:27.680+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-03-12T11:13:27.707+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-03-12T11:13:27.722+0000] {sql.py:457} INFO - Running statement:  
            insert into dag_runs (dt, dag_id) values ('2024-03-12', 'dag_with_postgres_operator_v02' )
        ), parameters: None
[2024-03-12T11:13:27.727+0000] {taskinstance.py:2728} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 282, in execute
    output = hook.run(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 404, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 462, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.SyntaxError: syntax error at or near ")"
LINE 3:         )
                ^

[2024-03-12T11:13:27.745+0000] {taskinstance.py:1149} INFO - Marking task as UP_FOR_RETRY. dag_id=dag_with_postgres_operator_v02, task_id=insert_into_table, execution_date=20240312T111325, start_date=20240312T111327, end_date=20240312T111327
[2024-03-12T11:13:27.772+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 889 for task insert_into_table (syntax error at or near ")"
LINE 3:         )
                ^
; 2395)
[2024-03-12T11:13:27.820+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-12T11:13:27.857+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
